src/main/java/uk/gov/ons/fwmt/resource_service/exception/ExceptionCode.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ons.fwmt.resource_service.exception;
+
+public enum ExceptionCode {
+  FWMT_JOB_SERVICE_0001("UNKNOWN"),
```

Comment:
Errorcodes should mention resource service

Reply:
its in the resource service logs what benefit do we get by mentioning that it is the resource service? This is the code that Chris is moving to the common service that he is waiting for you to review so that change would be more appropriate there

Reply:
Ok, we never log into CF to see the logs, we use centralised tool to monitor the logs.
For e.g.: In Splunk we could trigger alert based on certain error code .
In order differentiate the ErrorCode per service and operation reason we use separate codes per service.

Chris code shouldn't move the error codes as such.  we need change certain things in Chris common jar. Once Live is through I will look into that.